specification/compute/resource-manager/Microsoft.Compute/preview/2018-06-01/gallery.json

```diff
@@ -0,0 +1,1120 @@
+            "name": "OperatingSystemStateTypes",
+            "modelAsString": false
+          }
+        },
+        "endOfLifeDate": {
+          "type": "string",
+          "format": "date",
+          "description": "The end of life of this gallery image."
+        },
+        "identifier": {
+          "$ref": "#/definitions/GalleryImageIdentifier"
+        },
+        "recommended": {
+          "$ref": "#/definitions/RecommendedMachineConfiguration"
+        },
+        "disallowed": {
+          "$ref": "#/definitions/Disallowed"
+        },
+        "purchasePlan":{
+          "$ref": "#/definitions/ImagePurchasePlan"
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The provisioning state, which only appears in the response."
+        }
+      },
+      "description": "Describes the properties of a gallery image."
+    },
+    "GalleryImageIdentifier": {
+      "properties":{
+        "publisher": {
+          "type": "string",
+          "description": "The gallery image publisher name."
+        },
+        "offer": {
+          "type": "string",
+          "description": "The gallery image offer name."
+        },
+        "sku": {
+          "type": "string",
+          "description": "The gallery image sku name."
+        }
+      },
+      "description": "This is the gallery image identifier."
+    },
+    "RecommendedMachineConfiguration": {
+      "properties": {
+        "vCPUs": {
+          "$ref": "#/definitions/ResourceRange"
+        },
+        "memory": {
+          "$ref": "#/definitions/ResourceRange"
+        }
+      },
+      "description": "Describes the recommended machine configuration."
+    },
+    "ResourceRange": {
+      "properties": {
+        "min": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The minimum number of the resource."
+        },
+        "max": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The maximum number of the resource."
+        }
+      },
+      "description": "Describes the resource range."
+    },
+    "Disallowed": {
+      "properties": {
+        "diskTypes": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "A list of disk types."
+        }
+      },
+      "description": "Describes the disallowed disk types."
+    },
+    "ImagePurchasePlan": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The plan ID."
+        },
+        "publisher": {
+          "type": "string",
+          "description": "The publisher ID."
+        },
+        "product": {
+          "type": "string",
+          "description": "The product ID."
+        }
+      },
+      "description": "Describes the gallery image purchase plan. This is used by marketplace images."
+    },
+    "GalleryImageVersion": {
```

Comment:
Is this a tracked resource? It is doing an allOf on a Resource which contains tags and location properties. A proxy resource does not have these. If it is indeed a tracked resource, PATCH must be supported.

Reply:
This is a tracked resource.

Reply:
please add Patch support